releasenotes/notes/make-macos-install-script-favor-arch-specific-dmgs-b59b3f209c462400.yaml

```diff
@@ -0,0 +1,12 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+enhancements:
+  - |
+    Make macOS installation script favor architecture-specific DMGs,
+    when available.
```

Comment:
As these are distributed separately from the channel, and are therefore not tied to Agent releases, I think we should skip the changelog entry in this case.